diff --git a/src/yyjson.c b/src/yyjson.c
index 16ecbac..3ca262c 100644
--- a/src/yyjson.c
+++ b/src/yyjson.c
@@ -922,6 +922,17 @@ static_inline FILE *fopen_safe(const char *path, const char *mode) {
     FILE *file = NULL;
     if (fopen_s(&file, path, mode) != 0) return NULL;
     return file;
+#elif defined(_WIN32)
+    // Convert UTF-8 path and mode to UTF-16
+    wchar_t wpath[MAX_PATH];
+    wchar_t wmode[8]; // "rb+", etc.
+
+    if (!MultiByteToWideChar(CP_UTF8, 0, path, -1, wpath, MAX_PATH)) return NULL;
+    if (!MultiByteToWideChar(CP_UTF8, 0, mode, -1, wmode, 8)) return NULL;
+
+    FILE *file = NULL;
+    if (_wfopen_s(&file, wpath, wmode) != 0) return NULL;
+    return file;
 #else
     return fopen(path, mode);
 #endif
