diff --git a/src/yyjson.c b/src/yyjson.c
index 16ecbac..01aa9cf 100644
--- a/src/yyjson.c
+++ b/src/yyjson.c
@@ -923,7 +923,7 @@ static_inline FILE *fopen_safe(const char *path, const char *mode) {
     if (fopen_s(&file, path, mode) != 0) return NULL;
     return file;
 #else
-    return fopen(path, mode);
+    return g_fopen(path, mode);
 #endif
 }
 
diff --git a/src/yyjson.h b/src/yyjson.h
index 88cec2b..5999d4d 100644
--- a/src/yyjson.h
+++ b/src/yyjson.h
@@ -41,6 +41,7 @@
 #include <limits.h>
 #include <string.h>
 #include <float.h>
+#include <glib.h>
 
 
 
